_posts/2022-01-05-cockpit-260.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Cockpit 260
+author: garrett
+date: '2022-01-05'
+tags: cockpit, podman
+slug: cockpit-260
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 260 and cockpit-podman 39:
+
+
+## bridge: Warning on missing cockpit-system package
```

Comment:
Let's reorder this a bit: I'd say the "Action required" first, then the two Client improvements, then the minor stuff

Reply:
Agreed; the order should be changed.